From 7533f192ac36c230ad5965a10d0e33800f59f8fb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Wed, 19 Mar 2014 14:39:55 +0100
Subject: [PATCH] Export OBJ_txt2nid()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Port from 1.43 to 1.35. It has already existed in different since 1.37.

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 SSLeay.xs | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/SSLeay.xs b/SSLeay.xs
index 5bec132..c6a1e66 100644
--- a/SSLeay.xs
+++ b/SSLeay.xs
@@ -2745,5 +2745,20 @@ const EVP_MD *EVP_sha256()
 
 #endif
 
+#if OPENSSL_VERSION_NUMBER < 0x0090700fL
+#define REM14 "NOTE: before 0.9.7"
+
+int
+OBJ_txt2nid(s)
+    char *s
+
+#else
+
+int
+OBJ_txt2nid(s)
+    const char *s
+
+#endif
+
 
 #define REM_EOF "/* EOF - SSLeay.xs */"
-- 
1.8.5.3

